src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8644,6 +8644,118 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/rule-sets':
+    get:
+      x-alfresco-since: "7.3.0"
+      tags:
+        - rules
+      summary: List rule-sets on a folder
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.3 and newer version.
+
+        Gets a list of rule sets that are used by the folder **nodeId**.
+
+        Currently, this list may only ever contain zero or one rule set entries as folders cannot use multiple rule sets.
```

Comment:
Trivial: re: "folders cannot use multiple rule sets" - is it "use" or something slightly different, eg. "own" (which I see is the term used further below) or else maybe "define" or "support" ?

Reply:
"use" was supposed to be a catch-all for both "own" and "linked", as a folder for the initial implementation cannot define its own rules at the same time as linking to another folder